sidebars.js

```diff
@@ -132,8 +132,16 @@ module.exports = {
       "smart-contracts/opshin",
       "smart-contracts/plu-ts",
     ],
-    "Be Part of the Governance": [
+    "Participate in Governance": [
       "governance/overview",
```

Comment:
Did we want to remove this? as it's already in Cardano Governance sub menu?

Reply:
@rdlrt where are you seeing it duplicated?  When this PR builds locally it shows only `Overview` and `Governance Actions` in the `Cardano Governance` submenu.

Reply:
Ah - my bad, I got confused with governance/overview and governance/cardano-governance/overview indeed